infra/mock_api/template/ecs.tf

```diff
@@ -0,0 +1,130 @@
+# fargate config goes in this file
+# create service and schedule (optional)
+
+resource "aws_ecr_repository" "mock-api-repository" {
+  name                 = "mock-api-repo"
+  image_tag_mutability = "IMMUTABLE"
```

Comment:
you'll probably want this to be MUTABLE for now to match the eligibility screener settings. Especially if you plan to use the `latest` in the short term!